graphql/tests/image/distros/data.json

```diff
@@ -0,0 +1,27 @@
+{
+    "image": [
+        {
+            "_id": "ubuntu1604"
+        }
+    ],
```

Comment:
I have a feeling that this particular data entry has no effect and that you could remove it